cmd/ttn-lw-cli/commands/applications.go

```diff
@@ -41,6 +41,9 @@ func applicationIDFlags() *pflag.FlagSet {
 }
 
 var errNoApplicationID = errors.DefineInvalidArgument("no_application_id", "no application ID set")
+var errNoConfirmation = errors.DefineInvalidArgument("no_confirmation", "action not confirmed")
+
+var applicationPurgeWarning = "This action will permanently delete the application and all related data (API keys, rights, attributes etc.)"
```

Comment:
Idiomatic would be to group them in a `var ( ... )` block

Reply:
BTW, are we using vars for messages like these elsewhere in the CLI?

If so, they would be `const`s, but I don't think we do this

Reply:
I haven't seen it used like this anywhere else.

Reply:
I grouped it inside one file but let me know if there is a better way of doing this.